src/commands/deploy.rs

```diff
@@ -0,0 +1,249 @@
+use anyhow::{Context, Result};
+use bindle::Id;
+use core::panic;
+use hippo_openapi::apis::{
+    account_api::api_account_createtoken_post,
+    app_api::{api_app_post, ApiAppPostError},
+    channel_api::api_channel_post,
+    configuration::{ApiKey, Configuration},
+    revision_api::{api_revision_post, ApiRevisionPostError},
+    Error,
+};
+use hippo_openapi::models::{
+    ChannelRevisionSelectionStrategy, CreateAppCommand, CreateChannelCommand, CreateTokenCommand,
+    RegisterRevisionCommand,
+};
+use reqwest::header;
+use std::path::PathBuf;
+use structopt::{clap::AppSettings, StructOpt};
+
+const APP_CONFIG_FILE_OPT: &str = "APP_CONFIG_FILE";
```

Comment:
I know you're following the existing pattern here, but do we know why we define these `const`s with basically matching names/strings to use them only once?

Reply:
We've mostly used the consts where there are interdependencies between arguments e.g. `conflicts_with` or `requires`, so that we can reference the related arg in a checked way.  (There are cases where the arg itself doesn't have interdependencies but it's in a command with other args that do, and in those cases it's been "so things look the same."  E.g. the log directory in `UpCommand`.  Also cases where the same arg appears under multiple commands and we want to ensure they have the same name in both places.)  If there's no dependencies and the names are just UI elements then it's probably not needed.

This particular one might fall under "stay consistent" as the same arg appears in UpCommand and the Bindle commands.  We don't currently have a file of common names/args though so we just replicate the consts and arg settings around.  Learn more in episode 14 of my long-running podcast Why Structopt Is Wrong.

...but that is completely orthogonal to this PR and to whether we should use the const here.  Sorry Michelle.